README.md

```diff
@@ -28,6 +28,8 @@ This repository contains algorithms and data structures implemented in C# for ed
 			* [Binary GCD](./Algorithms/Numeric/GreatestCommonDivisor/BinaryGreatestCommonDivisorFinder.cs)
 		* [Factorization](./Algorithms/Numeric/Factorization)
 		    * [Trial division](./Algorithms/Numeric/Factorization/TrialDivisionFactorizer.cs)
+		* [Series](./Algorithms/Numeric/Series/)
+	    	* [Maclaurin](./Algorithms/Numeric/Series/Maclaurin.cs)
```

Comment:
```suggestion
		* [Maclaurin](./Algorithms/Numeric/Series/Maclaurin.cs)
```
Thanks for the contribution, I'll review and merge this PR a bit later as I'm rather busy at the moment